WcaOnRails/app/controllers/api/internal/v1/payment_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class Api::Internal::V1::PaymentController < Api::Internal::V1::ApiController
+  def init
+    competition_id, user_id = params["attendee_id"].split("-")
```

Comment:
This kind of parsing logic could move into a dedicated `Attendee` monolith model.